module_evaluations/TCR-50_025_01-02-ui-linked-data.MD

```diff
@@ -0,0 +1,63 @@
+* [x] Module build MUST produce a valid module descriptor
+  * - module descriptor provided by `ui-ld-folio-wrapper`.
+* [x] Module descriptor MUST include interface requirements for all consumed APIs
+  * - module descriptor provided by `ui-ld-folio-wrapper` includes the consumed APIs, see `ui-ld-folio-wrapper/package.json`.
+* [x] Third party dependencies use an Apache 2.0 compatible license
+* [x] Installation documentation is included
+  * -_note: read more at https://github.com/folio-org/mod-search/blob/master/README.md_
+* [x] Personal data form is completed, accurate, and provided as `PERSONAL_DATA_DISCLOSURE.md` file
+* [x] Sensitive and environment-specific information is not checked into git repository
+  * - according to the reviewer's good-faith effort
+* [x] Module is written in a language and framework from the [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) page[^1]
+* [x] Module only uses FOLIO interfaces already provided by previously accepted modules _e.g. a UI module cannot be accepted that relies on an interface only provided by a back end module that hasn't been accepted yet_
+  * - `mod-linked-data` currently under review, [TCR-46](https://folio-org.atlassian.net/browse/TCR-46).
+* [x] Module gracefully handles the absence of third party systems or related configuration
+  * - If the module cannot load a list of possible values from the LoC APIs, it displays an error message about being unable to load the data.
+  * - It will use a cached copy of of the values, if available.
+* [x] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication (6); and any disabled or intentionally ignored rules/recommendations are reasonably justified.
+  * See [Rule Customization](https://dev.folio.org/guides/code-analysis/#rule-customization) details. 
+* [x] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools[^1]
+ * - uses node
+* [x] Unit tests have 80% coverage or greater, and are based on [officially supported technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1]
+
+## Frontend
+* [x] If provided, End-to-end tests must be written in an [officially supported technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)[^1]
+  * -_note: while it's strongly recommended that modules implement integration tests, it's not a requirement_
+  * -_note: these tests are defined in https://github.com/folio-org/stripes-testing_
+* [x] Have i18n support via react-intl and an `en.json` file with English texts
+* [ ] Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension
+  * - Errors caught in self-eval have been fixed.
+  * - Two errors caught by WebAIM, a broken ARIA menu and two empty table headers in the results page over the first and last columns (selection buttons and Edit instance buttons)
```

Comment:
These have been reported to the team. 
Despite the error given by WAVE, I've been unable to identify exactly where the broken ARIA menu.
The empty table headers are bad practice, but I'm not certain what effect they have in context or what label would be meaningful.

Reply:
> These have been reported to the team.

Has the team committed to fix these in some reasonable timeframe?  Or if they are so complex that they require significant research (without seeing the code I doubt this is true for the table headers, but *maybe* for the aria issue?), have they at least committed to a spike?  If so then I think provisional acceptance would be reasonable.

Reply:
As of this morning, they are reported to be fixed by https://github.com/folio-org/ui-linked-data/pull/79/files